packages/calendar/src/Calendar.ts

```diff
@@ -0,0 +1,511 @@
+                    ${this.weekdays.map((weekday) =>
+                        this.renderWeekdayColumn(weekday)
+                    )}
+                </tr>
+            </thead>
+        `;
+    }
+
+    public renderWeekdayColumn(weekday: CalendarWeekday): TemplateResult {
+        return html`
+            <th
+                role="columnheader"
+                scope="col"
+                class="spectrum-Calendar-tableCell"
+            >
+                <abbr class="spectrum-Calendar-dayOfWeek" title=${weekday.long}>
+                    ${weekday.narrow}
+                </abbr>
+            </th>
+        `;
+    }
+
+    public renderCalendarTableBody(): TemplateResult {
+        return html`
+            <tbody role="presentation">
+                ${this.weeksInCurrentMonth.map((weekIndex) =>
+                    this.renderCalendarTableRow(weekIndex)
+                )}
+            </tbody>
+        `;
+    }
+
+    public renderCalendarTableRow(weekIndex: number): TemplateResult {
+        return html`
+            <tr role="row">
+                ${this.getDatesInWeek(weekIndex).map((calendarDate) =>
+                    this.renderCalendarTableCell(calendarDate)
+                )}
+            </tr>
+        `;
+    }
+
+    public renderCalendarTableCell(calendarDate: CalendarDate): TemplateResult {
+        const isOutsideMonth = calendarDate.month !== this.currentDate.month;
+
+        const isSelected = Boolean(
+            this.selectedDate &&
+                isSameDay(this.toCalendarDate(this.selectedDate), calendarDate)
+        );
+
+        const isToday = isSameDay(calendarDate, this.today);
+
+        const isDisabled = Boolean(
+            this.disabled ||
+                (this.minDate && calendarDate.compare(this.minDate) < 0) ||
+                (this.maxDate && calendarDate.compare(this.maxDate) > 0)
+        );
+
+        const dayClasses = {
+            'is-outsideMonth': isOutsideMonth,
+            'is-selected': isSelected,
+            'is-today': isToday,
+            'is-disabled': isDisabled,
+        };
+
+        // TODO: The title must include "Today," and " selected" translated to the current language
+        const currentDayTitle = this.formatDate(calendarDate, {
+            weekday: 'long',
+            year: 'numeric',
+            month: 'long',
+            day: 'numeric',
+        });
+
+        return html`
+            <td
+                role="gridcell"
+                class="spectrum-Calendar-tableCell"
+                title=${currentDayTitle}
+                tabindex=${ifDefined(!isOutsideMonth ? '-1' : undefined)}
+                aria-disabled=${isOutsideMonth || this.disabled}
+                aria-selected=${isSelected}
+            >
+                <span
+                    role="presentation"
+                    class="spectrum-Calendar-date ${classMap(dayClasses)}"
+                    @click=${() => this.handleDayClick(calendarDate)}
```

Comment:
There is a small perf win to be had to cache this call to `handleDayClick()` in a way that would allow us to pass the same method here each time. The anonymous method will be added and removed as a listener in each render.